iina/MainWindowController.swift

```diff
@@ -728,9 +728,9 @@ class MainWindowController: PlayerWindowController {
         titleBarHeightConstraint.constant = TitleBarHeightWithOSC
       }
       // Remove this if it's acceptable in 10.13-
-      // titleBarBottomBorder.isHidden = true
+      titleBarBottomBorder.isHidden = true
```

Comment:
I'm not understanding how this change is related.

Reply:
Wasn't sure what the intent was there. The bottom sidebar doesn't have a border, and I did think the top sidebar looked better without a border, especially when the side panel was open. But this is not the hill I want to die on. Backed out the change in the new push.